tests/scripts/custodian/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM ubuntu:20.04
```

Comment:
Just a platform question.

Why Ubuntu and not the SUSE BCI, in case this eventually ends in the rancher docker registry?

Reply:
updated to use suse bci registry